pages/specification/json-hyper-schema/_index.md

```diff
@@ -13,6 +13,531 @@ In essence:
+
+#### Submission
+
+Using a link to submit user input. (Analogous to `<form method="post">`)
+
+```json
+{
+  "links": [
+    {
+      "href": "/books"
+      "rel": "collection",
+      "submissionSchema": {
+        "type": "object",
+        "properties": {
+          "title": { "type": "string" },
+          "authorId": { "type": "integer" },
+          ...
+        },
+        "required": ["title", "authorId", ...]
+      }
+    }
+  ]
+}
+```
+
+```http
+POST https://example.com/books
+Content-Type: application/json
+
+{
+  "title": "Moby-Dick",
+  "authorId": 100,
+  ...
+}
+```
+
+_This `POST` request submits data to create a new book with the title "Moby-Dick" and `authorId` 100._
+
+```http
+204 No Content
+```
+
+_The request was successful, and the book was created, but the response contained no content._
+
+##### Submission Media Type
+
+You can also make submission will media types other than JSON.
+
+```json
+{
+  "links": [
+    {
+      "href": "/books"
+      "rel": "collection",
+      "submissionMediaType": "application/xml",
+      "submissionSchema": {
+        "type": "object",
+        "properties": {
+          "title": { "type": "string" },
+          "authorId": { "type": "integer" },
+          ...
+        },
+        "required": ["title", "authorId", ...]
+      }
+    }
+  ]
+}
+```
+
+```http
+POST https://example.com/books
+Content-Type: application/xml
+
+<Book>
+  <title>Moby-Dick<title>
+  <authorId>100</authorId>
+  ...
+</Book>
+```
+
+```http
+204 No Content
+```
+
+_This code shows using application/xml, to create a new book resource with a `POST` request._
+
+### Relations
+
+#### Registered relation
+
+A registered relation pointing to an author.
+
+```json
+{
+  "links": [
+    {
+      "href": "https://example.com/authors/100",
+      "rel": "author"
+    }
+  ]
+}
+```
+
+#### Custom relation
+
+A custom relation linking to a specific review.
+
+```json
+{
+  "links": [
+    {
+      "href": "https://example.com/review/6789",
+      "rel": "https://example.com/relations/review"
+    }
+  ]
+}
+```
+
+#### Multiple relations
+
+A multiple relations associating a comments collection with two different relationship types.
+
+```json
+{
+  "links": [
+    {
+      "href": "https://example.com/reviews",
+      "rel": ["collection", "https://example.com/relations/reviews"]
+    }
+  ]
+}
+```
+
+### Base URIs
+
+A base URI is used to resolve relative URIs.
+
+#### Default
+
+By default, the base URI for relative URIs in LDOs is the URI used to retrieve the resource
+
+```http
+GET https://example.com/books/1
+```
+
+```json
+{
+  "links": [
+    {
+      "href": "/author/1",
+      "rel": "author"
+    }
+  ]
+}
+```
+
+Link Target: `https://example.com/author/1`
+
+#### Setting the base URI
+
+`base` can be used to alter the base URI. It is a URI Template and can be relative.
```

Comment:
suggestion: remove uppercase from "Template"

Google developer style: https://developers.google.com/style/capitalization

Reply:
URI Template is a proper name referring to [RFC 6570](https://www.rfc-editor.org/rfc/rfc6570). It should be capitalized.